pkg/apis/core/v1/validation/validation.go

```diff
@@ -107,7 +107,8 @@ func ValidateNonnegativeQuantity(value resource.Quantity, fldPath *field.Path) f
 }
 
 // Validate compute resource typename.
-// Refer to docs/design/resources.md for more details.
+// Refer to https://github.com/kubernetes/community/blob/01036e0d3f4230b2ba8affbb151ddd58d4706aa3/contributors/design-proposals/scheduling/resources.md
```

Comment:
should this refer to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ instead?

Reply:
I am afraid this page doesn't contain exact rule like [this section](https://github.com/kubernetes/community/blob/01036e0d3f4230b2ba8affbb151ddd58d4706aa3/contributors/design-proposals/scheduling/resources.md#resource-types)